packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -75,6 +75,10 @@ export class TableLayout<T> extends ListLayout<T> {
     super.validate(invalidationContext);
   }
 
+  // TODO: in the RAC case, we don't explicity accept loadingState on the TableBody, but note that if the user
+  // does happen to set loadingState="loading" or loadingState="loadingMore" coincidentally, the isLoading
+  // part of this code will trigger and the layout will reserve more room for the loading spinner which we actually only use
+  // in RSP
   protected buildCollection(): LayoutNode[] {
```

Comment:
Perhaps will change if we split out all of the spectrum specific logic into a separate implementation